geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb

```diff
@@ -27,30 +27,31 @@ Configure Portable Data eXchange for all the cache(s) in the cluster.
 ## <a id="topic_jdkdiqbgphqh" class="no-quick-link"></a>configure pdx
 
 <a id="topic_jdkdiqbgphqh__section_C27BE964CE554180A65968DBEBF50B23"></a>
-Configures <%=vars.product_name%>'s Portable Data eXchange for all the cache(s) in the cluster. This command does not effect on the running members in the system. This command persists the pdx configuration in the locator with cluster configuration service.
 
-**Note:**
-This command should be issued before starting any data members.
+Configures <%=vars.product_name%>'s Portable Data eXchange for all the cache(s) in the cluster
+and persists the pdx configuration in the locator with the cluster configuration service.
+
+This command does not affect the running members in the system, so should be issued before starting any data members.
```

Comment:
I think there's a word missing here. Maybe either "so it should be issued" or "and so should be issued."